scipy/stats/_stats_py.py

```diff
@@ -10244,9 +10463,17 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     References
     ----------
     .. [1] "Wasserstein metric", https://en.wikipedia.org/wiki/Wasserstein_metric
```

Comment:
Would be good to refer to this in the text if it's being cited. It could go after the first sentence at the top.